components/places/src/db/schema.rs

```diff
@@ -174,6 +184,7 @@ fn upgrade(db: &PlacesDb, from: i64) -> Result<()> {
     )?;
     migration(db, 4, 5, &[CREATE_SHARED_SCHEMA_SQL], || Ok(()))?;
     migration(db, 5, 6, &[CREATE_SHARED_SCHEMA_SQL], || Ok(()))?; // new tags tables.
+    migration(db, 6, 7, &[CREATE_SHARED_SCHEMA_SQL], || Ok(()))?; // bookmark syncing.
```

Comment:
I do feel as though 'rerun the whole schema' is going to eventually prove problematic, but we can cross that bridge then, I guess.

Reply:
Filed #849 so we don't lose track of this.